RiotSharp/Endpoints/TftMatchEndpoint/TftMatch.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Newtonsoft.Json;
+using RiotSharp.Endpoints.MatchEndpoint;
+using RiotSharp.Misc.Converters;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RiotSharp.Endpoints.TftMatchEndpoint
+{
+    public class TftMatch
```

Comment:
This class does not match the schema of Riot's api resonse. You might have forgotten to include your changes for this one.